src/helpers/authManager.js

```diff
@@ -121,6 +121,9 @@ const initAuthMgr = (bInit) => {
     authConfig.userIdentifier = sdkConfigAuth.mashupUserIdentifier;
     authConfig.password = sdkConfigAuth.mashupPassword;
   }
+  if( sdkConfigAuth.iframeLoginUI ){
```

Comment:
Suggest looking for this optional setting and coping with it being a boolean of true or the string "true"